config/urls.py

```diff
@@ -14,8 +14,9 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('', include('school.urls')),
```

Comment:
Do not write include for '' cause it triggers on every path. Instead include 'school/' and then "update", "create"